valohai.yaml

```diff
@@ -30,8 +30,8 @@
     name: batch-inference
     image: tensorflow/tensorflow:2.6.0
     command:
-    - pip install pillow valohai-utils
-    - python ./batch_inference.py
+      - pip install pillow valohai-utils
+      - python ./batch_inference.py
```

Comment:
As discussed earlier, these changes should excluded from this PR and be another discussion if they are necessary.